public/config/locales/de.yml

```diff
@@ -294,6 +294,7 @@ de:
       thumbnail: Vorschaubild
       link: Verweis zu digitalem Objekt
       badge_label: Digital %{level}
+      go: Gehen Sie zur Datei
```

Comment:
@brianzelip Weblate should take care of the non-English languages. No need to do them yourself.